client/src/main/scala/org/apache/celeborn/client/WorkerStatusTracker.scala

```diff
@@ -198,6 +201,6 @@ class WorkerStatusTracker(
     shuttingWorkers.retainAll(newShutdownWorkers)
     val shutdownList = newShutdownWorkers.asScala.filterNot(shuttingWorkers.asScala.contains).asJava
     shuttingWorkers.addAll(newShutdownWorkers)
-    shutdownList
+    newShutdownWorkers
```

Comment:
If so, we can simplify this code.

Reply:
This function's origin meaning is that only newly added shutdownworkers can be returned (which is not in shuttingWorkers ), but now it returns all the shutdownworkers. Flink's shufflemaster will register ShuffleResourceTracker as workerStatusListeners. ShuffleResourceTracker should juse processes the newly add shutdownworkers. Is that right ? @RexXiong

Reply:
ShuffleResourceTracker of Flink doesn't care shutdown workers currently @zhongqiangczq